preliminary-work/nematic_order.py

```diff
@@ -0,0 +1,24 @@
+import flowermd
+import hoomd
+import gsd
+import numpy as np
+import freud
+
+
+def nematic_order(gsd_file):
+
+    bond_vectors = []
```

Comment:
We need to move this list below the `for snap in traj` for loop.  For each snapshot, we want a new list of bond vectors being used to calculate the nematic order for that snapshot only.